drydock_provisioner/drivers/node/maasdriver/api_client.py

```diff
@@ -43,6 +43,8 @@ class MaasRequestFactory(object):
     def __init__(self, base_url, apikey):
         self.base_url = base_url
         self.apikey = apikey
+
+        print("Creating MaaS API client for URL %s with key %s" % (base_url, apikey))
```

Comment:
This looks like a testing relic. As it's the API client, this should probably be a LOG statement instead.